### ISSUE TITLE
Slim includes don't work after update

### ISSUE THREAD

**Issue by drgullin** [bdkjones/CodeKit2#458]
v2.1.8 (18027)

```
/ options
include partials/_options.slim

/ header
include partials/_header.slim

ul.breadcrumbs
```

compiles to:

```
<include>partials/_options.slim</include><include>partials/_header.slim</include>
<ul class="breadcrumbs">
</ul>
```

log:

```
index.slim
File compiled successfully.
```

any workaround?

**Comment by bdkjones** [comment]
Yep. The `include` module was made an official part of Slim with 2.1, which is what's in the latest release of CodeKit. There's an issue with it. I'll have a fix for it out later today.

**Comment by bdkjones** [comment]
Update: I'm just waiting on a release of Zurb Foundation that's supposed to be coming this week. When that happens, I'm going to ship 2.1.9 which resolves this issue. (2.1.9 includes Libsass 3.0, which won't compile Zurb Foundation correctly until Zurb updates.)

**Comment by bdkjones** [comment]
Fixed for 2.1.9. STILL waiting on Zurb update. If it's not out tomorrow, I'm going to ship anyway.

**Comment by drgullin** [comment]
Is this option available in 2.1.9?

**Comment by bdkjones** [comment]
Yep.
